### DISCUSSION TITLE
天翼云盘挂载报错了

### DISCUSSION THREAD

**Discussion by yoyo3287258** [AlistGo/alist#3473] [General]
报错信息如下：
Failed init storage: Post "https://api.nn.ci/ocr/file/json": x509: certificate signed by unknown authority

**Comment by yoyo3287258** [comment]
定位到问题了，勾选“不使用OCR”，问题解决了

**Reply by ahazhang** [reply]
我也遇到这个报错了，但是没看到“不使用OCR”这个选项啊

**Reply by ahazhang** [reply]
解决了，原来需要选择  天翼云盘客户端，我之前一直选的是   天翼云盘

**Reply by ppp1974** [reply]
> 定位到问题了，勾选“不使用OCR”，问题解决了

已经解决了，谢谢，重新打开浏览器有 OCR选项了。

**Comment by youtehub** [comment]
最新版   **v3.11.0**  中的解决办法是，将  **https**  改为  **http** , 修改的位置如图所示
建议打开这个选项，不然每次重新启动 **alist** 需要 **手动输入验证码**,或者更换成你自己部署的 ocr 验证码识别服务
![image](https://user-images.githubusercontent.com/18629921/219908528-6415016c-4e20-472e-a842-0722eb455a99.png)

**Reply by yoyo3287258** [reply]
> 最新版 **v3.11.0** 中的解决办法是，将 **https** 改为 **http** , 修改的位置如图所示 建议打开这个选项，不然每次重新启动 **alist** 需要 **手动输入验证码**,或者更换成你自己部署的 ocr 验证码识别服务 ![image](https://user-images.githubusercontent.com/18629921/219908528-6415016c-4e20-472e-a842-0722eb455a99.png)

感谢

**Comment by youtehub** [comment]
这个随着部署alist挂载天翼云盘的人越来越多，使用的人也越来越多，如果是有自己的服务器的，最好还是自己部署一个。docker命令作者也提供了【docker run -d --restart=always -p 9898:9898 xhofe/ddddocr_server:main】

接下来的操作：https://alist.nn.ci/zh/faq/howto.html

**Reply by yoyo3287258** [reply]
部署这个之后，orc接口的url就填localhost:9898 吗

**Reply by youtehub** [reply]
看看这篇文档吧，https://alist.nn.ci/zh/faq/howto.html
